src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -117,132 +117,30 @@
 # import boto sdb connection for autoscaling
 import boto.sdb
 
+import textwrap
```

Comment:
Nit: I prefer to keep imports in a sorted order, as this makes it easier to track the diff of imports over time.